cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var ajax = require( 'xdr' );
```

Comment:
FYI @sebworks I'll replace this third party xhr library with [ours](https://github.com/cfpb/cfgov-refresh/blob/6b7f7d005a0ce189e84151a31801077dc76e7897/cfgov/unprocessed/js/modules/util/ajax-request.js) in a future PR.